lib/cusparse/array.jl

```diff
@@ -17,12 +18,12 @@ Base.convert(T::Type{<:AbstractCuSparseArray}, m::AbstractArray) = m isa T ? m :
 mutable struct CuSparseVector{Tv, Ti} <: AbstractCuSparseVector{Tv, Ti}
     iPtr::CuVector{Ti}
     nzVal::CuVector{Tv}
```

Comment:
It would be nice if this was `nzval` like Base sparse has. That would make general code work.

Reply:
It's unlikely any generic code is actually going to work, but sure.